_posts/2021-02-24-#21224.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: pr
+date: 2021-02-24
+title: "Halt processing of unrequested transactions"
+pr: 21224
+authors: [ariard]
+components: ["p2p"]
+host: ellemouton
+status: upcoming
+commit:
+---
+
+## Notes
```

Comment:
I think these could be a bit more detailed. For example, it wouldn't do any harm to repeat the transaction relay message flow. We've done this before, eg in the review clubs for 18238 and 15505, but it's good to repeat it for newcomers and reinforce it for others.